stack-8.6.1.yaml

```diff
@@ -0,0 +1,6 @@
+resolver: nightly-2018-10-01
+
+extra-deps:
+  - base-noprelude-4.12.0.0
+  - relude-0.3.0
```

Comment:
Actually, `relude-0.3.0` is already in this nightly snapshot, so you can safely remove it from `extra-deps`

Reply:
Done!